src/nowarn/lauxlib.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#if defined(__clang__)
+    #pragma clang diagnostic push
+    #pragma clang diagnostic ignored "-Wold-style-cast"
+    #include "lua/lauxlib.h"
+    constexpr int lual_buffersize = LUAL_BUFFERSIZE;
+    #pragma clang diagnostic pop
+#elif defined(__GNUG__)
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wold-style-cast"
+    #include "lua/lauxlib.h"
+    constexpr int lual_buffersize = LUAL_BUFFERSIZE;
```

Comment:
I think for consistency with other symbols this should have a capital L: `luaL_buffersize`

Reply:
Fixed.